src/main/java/duke/DialogBox.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
```

Comment:
I think you might have forgotten to include javadocs for this class and some of the methods in this class too